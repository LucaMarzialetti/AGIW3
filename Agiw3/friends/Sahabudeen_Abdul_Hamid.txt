--url_fb https://www.facebook.com/sahabudeen.abdulhamid
--friends
VM42599:3 Hajji Mohmmed
VM42599:3 Mohamed Aslam
VM42599:3 ராஜூ. செல்வமணி
VM42599:3 Feroz Khan
VM42599:3 Vijayakumar Devarajan
VM42599:3 Srinagesh Burugucherla
VM42599:3 Raju Raju
VM42599:3 Abdul Rauf A R
VM42599:3 Mohd Darvis
VM42599:3 Sheik Kader
VM42599:3 Sikkandar Batcha
VM42599:3 Mansoor Kasim
VM42599:3 Tmmk Yassar
VM42599:3 Ali Jabarullah
VM42599:3 Mohammed Rafik Jackriya Rafik
VM42599:3 Lijo Oommen
VM42599:3 Hari Malayan
VM42599:3 Nasurudeen Thajudeen
VM42599:3 Jayabalan Gukanesan
VM42599:3 Mohamed Nazar
VM42599:3 Hameed Guthus
VM42599:3 Abdul Azees Ali
VM42599:3 Bashir Ahamed
VM42599:3 Syed Mubarak
VM42599:3 Faisal MrBeams
VM42599:3 Rafik Batcha
VM42599:3 Jacob Ravibalan
VM42599:3 Mohammed Gani
VM42599:3 Mohamed Sulthan
VM42599:3 Mohamed Ibrahim
VM42599:3 Jiaudeen W Jiaudeen
VM42599:3 Sameerul Hameed
VM42599:3 Mohamed Ali Jinnah
VM42599:3 Sahab Deen
VM42599:3 Haja Sheriff
VM42599:3 Abdulrauf Abdulrasheed
VM42599:3 Naze Fasil Sirajudeen
VM42599:3 Mohamed Refoizudeen Maricar
VM42599:3 Mohamed Saud
VM42599:3 Chelliah Murugesan
VM42599:3 Mohideen Abdul Kader
VM42599:3 Jefrudeen Bj
VM42599:3 Sakthivel Karup
VM42599:3 Joseph Antony Marri
VM42599:3 Basheer Ahamed
VM42599:3 Hari Hara Kumar
VM42599:3 Mohammad Ibrahim
VM42599:3 Abusuhoodu London
VM42599:3 Abdul Shajahan
VM42599:3 Haja Maricar
VM42599:3 Jafrudeen Jafar
VM42599:3 Mohamed Hussain
VM42599:3 Humaun Ali
VM42599:3 Saharjath Begum
VM42599:3 அய்மான் சங்கம்
VM42599:3 Elayaraja Paneer
VM42599:3 Alhana Begum
VM42599:3 Iyarthevan Muthu
VM42599:3 Murari Venkatachalapathi Santharaman
VM42599:3 Mohamed Abdul Jahangeer
VM42599:3 Mohamed Gazzali
VM42599:3 Hussain Basha
VM42599:3 Sam Zab