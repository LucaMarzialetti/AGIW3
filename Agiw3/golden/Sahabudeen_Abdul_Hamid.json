{
    "PATTERN": {
        "tel": [],
        "addr": [],
        "email": []
    },
    "NER": {
        "LOC": [
            "Kishoreganj",
            "PAHMC",
            "Bangladesh"
        ],
        "ORG": [
            "Abdul Hamid Medical College & Hospital",
            "Abdul Hamid Medical College",
            "Guru Dayal College",
            "Dhaka Central Law College",
            "Nikly GC High School",
            "Kishoreganj Bar"
        ],
        "PER": [
            "President Md Abdul Hamid",
            "President Abdul Hamid",
            "Rashida Hamid",
            "President",
            "Mithamoin Upazila"
        ]
    },
    "url": "http://www.ahmc.edu.bd/"
}